experimental/LieAlgebras/src/AbstractLieAlgebra.jl

```diff
@@ -143,8 +206,16 @@ function lie_algebra(
   return AbstractLieAlgebra{elem_type(R)}(R, struct_consts2, Symbol.(s); cached, check)
 end
 
+@doc raw"""
+    lie_algebra(R::Ring, dynkin::Tuple{Char,Int}; cached::Bool) -> AbstractLieAlgebra{elem_type(R)}
+
+Construct the Lie algebra over the ring `R` with Dynkin type given by `dynkin`.
```

Comment:
```suggestion
Construct the simple Lie algebra over the ring `R` with Dynkin type given by `dynkin`.
```

Ideally this would also describe what `dynkin` is exactly.

On the long term, I guess we want dedicated machinery to deal with (extended) Dynkin diagrams, (generalized) Cartan matrices, etc. etc., and then use these everywere. But baby steps... :-)